src/Materials/Textures/internalTexture.ts

```diff
@@ -111,14 +112,20 @@ export class InternalTexture {
     public url: string = "";
     /** @hidden */
     public _originalUrl: string; // not empty only if different from url
-    /**
-     * Gets the sampling mode of the texture
-     */
-    public samplingMode: number = -1;
     /**
      * Gets a boolean indicating if the texture needs mipmaps generation
      */
     public generateMipMaps: boolean = false;
+    /**
+     * Gets a boolean indicating if the texture uses mipmaps
+     * TODO implements useMipMaps as a separate setting from generateMipMaps

```

Comment:
Could you remind me why we wanted the split ?

Reply:
See #8863